packages/TelegramClient-Tests.package/TCTUITest.class/instance/assert.looksLike..st

```diff
@@ -0,0 +1,11 @@
+asserting
+assert: aMorph looksLike: aScreenshotFilename
+	
+	| screenshotHash morphHash |
+	
+	screenshotHash := self computeHashOf: (ImageReadWriter formFromFileNamed: aScreenshotFilename).
+	morphHash := self computeHashOf: (self takeScreenshotOf: aMorph).
+	
+	"only comparing hashes might lead to false positives"
+	screenshotHash = morphHash ifFalse: 
+			[AssertionFailure signal: 'Assertion failed']
```

Comment:
Maybe we can use an AssertionFailure that give the developer more useful information.

Reply:
Done